frontend/src/modules/Core/Constants/Theme.ts

```diff
@@ -209,6 +209,9 @@ theme = createTheme(theme, {
             borderColor: theme.palette.essentials[50],
           },
         },
+        textPrimary: {
+          color: theme.palette.purple[50],
+        },
```

Comment:
To add element-specific styling to emailSentAction in the email success snackbar, put this as a prop for the Button (don't need to style the Snackbar itself):
```js
sx={{ color: 'purple.50' }}
```

Reply:
Now I feel dumb trying to change the snackbar props lol